en-US/win10/SupportedInterfaces.md

```diff
@@ -54,8 +54,6 @@ Windows 10 IoT Core supports a variety of peripheral interfaces and protocols, i
 
 Browse, search, and filter, peripherals that are known to be supported on the **Raspberry Pi 2** and **MinnowBoard Max**. [Contribute to this list on GitHub.]
 
-This page makes use of JavaScript. If you have JavaScript disabled, search tools will be disabled.
-
 Looking for information about supported hardware platforms? Click [here](https://msdn.microsoft.com/library/windows/hardware/dn914597(v=vs.85).aspx) to be taken to a list of development boards compatible with Windows.
 
 <div class="searchbox">
```

Comment:
a JS warning is now shown in the searchbox if user doesn't have JavaScript, so I am removing the other, unneeded warning.